docs/br/mission.html

```diff
@@ -431,11 +431,10 @@ <h2 class="mb-2 text-2xl font-bold text-gray-200">
         class="flex flex-col-reverse mx-auto max-w-screen-3xl md:flex-row md:justify-between"
       >
         <section>
-          <a href="javascript: Cookiebot.renew()">Definições de cookies</a><br>
           <div>
-            <a target="_blank" href="../assets/privacy_policy.pdf"
-              >Política de Privacidade</a
-            >
+            <a target="_blank" href="cookie_policy.pdf">Política de cookies </a> |
+            <a href="javascript: Cookiebot.renew()">Definições de cookies</a>|
```

Comment:
```suggestion
            <a href="javascript: Cookiebot.renew()">Definições de cookies</a> |
```
can you please add a space here?

Reply:
for all